benchmarks/nHcal/nhcal_acceptance/config.yml

```
@@ -0,0 +1,40 @@
+sim:nhcal_acceptance:
+  extends: .det_benchmark
+  stage: simulate
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+        INDEX: ["00","01","02","03","04","05","06","07","08","09"]
+  script:
+    - snakemake --cores 5 sim_output/nhcal_acceptance/E${ENERGY}/sim_epic_backward_hcal_only.${INDEX}.edm4hep.root
+
+bench:nhcal_acceptance_combine:
+  extends: .det_benchmark
+  stage: benchmarks
+  needs: 
+    - "sim:nhcal_acceptance"
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+  script:
+    - snakemake --cores 5 sim_output/nhcal_acceptance/sim_epic_backward_hcal_only_E${ENERGY}_combined_10files.edm4hep.root
+
+
+bench:nhcal_acceptance_analysis:
+  extends: .det_benchmark
+  stage: benchmarks
+  needs:
+    - "bench:nhcal_acceptance_combine"
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+  script:
+    - snakemake --cores 3 results/nhcal_acceptance_analysis_epic_backward_hcal_only_E${ENERGY}_combined_10files.pdf
+
+collect_results:nhcal_acceptance:
+  extends: .det_benchmark
+  stage: collect
+  needs: 
+    - "bench:nhcal_acceptance_analysis"
+  script:
+    - ls -lrht
```

Each job is currently responsible to cleanup their respective simulation files as soon as possible.
```suggestion
    - ls -lrht
    - mv results{,_save}/ # move results directory out of the way to preserve it
    - for ENERGY in "1GeV" "5GeV" "10GeV"; do snakemake $SNAKEMAKE_FLAGS --cores 1 --delete-all-output results/nhcal_acceptance_analysis_epic_backward_hcal_only_E${ENERGY}_combined_10files.pdf; done
    - mv results{_save,}/
```